### ISSUE TITLE
Upgrading @babel/runtime to 7.0.0-beta.56 breaks the build

### ISSUE THREAD

**Issue by vladholubiev** [meteor/babel#22]
**What I tried:**
* Installing `@babel/runtime-corejs2`
* Adding custom `babelrc` with this: `{ "plugins": [ "@babel/plugin-transform-runtime", { "corejs": 2 } ] }`
* Installing `@babel/plugin-transform-runtime` with `@babel/core`  and pinning them to `7.0.0-beta.56`

But didn't change the situation.

**Relevant links:**

Relevant post on Meteor Forum:  https://forums.meteor.com/t/error-cannot-find-module-babel-runtime-helpers-builtin-interoprequiredefault/44944/7

Relevant Meteor issue: https://github.com/meteor/meteor/issues/10126

Babel release changelog: https://github.com/babel/babel/releases/tag/v7.0.0-beta.56

Updated docs for runtime plugin: https://github.com/babel/babel/releases/tag/v7.0.0-beta.56

Original PR in Babel: https://github.com/babel/babel/pull/8266

**Here is the stacktrace:**

```
=> Started proxy.                             
W20180804-14:25:34.341(0)? (STDERR) XXXXXXXXX/.meteor/local/build/programs/server/boot.js:475
W20180804-14:25:34.381(0)? (STDERR) }).run();
W20180804-14:25:34.381(0)? (STDERR)    ^
W20180804-14:25:34.382(0)? (STDERR) 
W20180804-14:25:34.382(0)? (STDERR) Error: Cannot find module '@babel/runtime/helpers/builtin/interopRequireDefault'
W20180804-14:25:34.382(0)? (STDERR)     at Function.Module._resolveFilename (module.js:547:15)
W20180804-14:25:34.383(0)? (STDERR)     at Function.resolve (internal/module.js:18:19)
W20180804-14:25:34.383(0)? (STDERR)     at Object.require (XXXXXXXXX/.meteor/local/build/programs/server/boot.js:288:32)
W20180804-14:25:34.384(0)? (STDERR)     at makeInstallerOptions.fallback (packages/modules-runtime.js:651:18)
W20180804-14:25:34.384(0)? (STDERR)     at require (packages/modules-runtime.js:244:16)
W20180804-14:25:34.384(0)? (STDERR)     at livedata_connection.js (XXXXXXXXX/.meteor/local/build/programs/server/packages/ddp-client.js:147:30)
W20180804-14:25:34.384(0)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:343:9)
W20180804-14:25:34.385(0)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180804-14:25:34.385(0)? (STDERR)     at namespace.js (packages/ddp-client/common/namespace.js:1:300)
W20180804-14:25:34.385(0)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:343:9)
```

**Comment by benjamn** [comment]
I will need to fix this when I update everything to `7.0.0-beta.56`. In the meantime, you might want to stay on `beta.55`. The good news is that Babel 7 seems very close to its first release candidate!

**Comment by pilarArr** [comment]
Babel 7 is out 🎉
